styles/partials/base-partials/_forms.scss

```diff
@@ -0,0 +1,195 @@
+//not sure this is needed
```

Comment:
Since everything we're doing is incompatible with existing quartz anyway, there's no harm in building up from nothing and tailoring specifically to these components